tests/stubs/devices/services/temperature.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
+
+import (
+	"fmt"
+	"github.com/paypal/gatt"
+)
+
+var dataWrite string
+var write bool
+var state string
+
+func NewTemperatureService() *gatt.Service {
+	temp := 36
+	s := gatt.NewService(gatt.MustParseUUID("09fc95c0-c111-11e3-9904-0002a5d5c51b"))
+
+	// Creating characteristic UUID for reading temperature
+	readCharacteriscticUUID := "11fac9e0-c111-11e3-9246-0002a5d5c51b"
+	//Adding a characteristic for reading temperature data
+	s.AddCharacteristic(gatt.MustParseUUID(readCharacteriscticUUID)).HandleReadFunc(
+		func(rsp gatt.ResponseWriter, req *gatt.ReadRequest) {
+			fmt.Fprintf(rsp, "%d", temp)
+		})
+
+	// Creating characteristic UUID for writing temperature
+	writeCharacteristicUUID := "16fe0d80-c111-11e3-b8c8-0002a5d5c51b"
+	// Adding characteristic for writing to device
+	s.AddCharacteristic(gatt.MustParseUUID(writeCharacteristicUUID)).HandleWriteFunc(
+		func(r gatt.Request, data []byte) (status byte) {
+			fmt.Println("Wrote:", string(data))
+			write = true
+			dataWrite = string(temp)
+			return gatt.StatusSuccess
+		})
+
+	// Creating characteristic UUID for reading data written to device
+	readwriteCharacteristicUUID := "1c927b50-c116-11e3-8a33-0800200c9a66"
```

Comment:
Can we use another variable name instead of "**readwriteCharacteristicUUID**" as this is not the same UUID as previously declared for this variable, and can cause confusion for the user

Reply:
Updated